### PR TITLE
Feature/kamsiyonna/work experiences

### PR DESCRIPTION
**Pull request by KamsiyonnaObi**
@brandonetter I accidentally committed all my changes without properly documenting them.

This feature is for the work exp section on mobile dev. I am working now on making it mobile-responsive.

### CODE REVIEW

File: components/ExperienceCard.tsx [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+            years_worked: string;
+        }[];
+    };
+}
+
+const ExperienceCard = (props: Props) => {
+  return (
+    <>
+        {props.experiences.data.map((exp, idx) => (
+            <Card key={idx} className="w-[345px] py-[36px] pl-[36px] rounded-[10px] gap-[18px] bg-white-900 shadow-work cursor-pointer sm:w-[607px] sm:border sm:border-white-800 sm:dark:border-none sm:shadow-none sm:hover:shadow-work sm:hover:ml-[30px] dark:bg-transparent hover:dark:bg-black-300">
```
Comment (brandonetter) [inline components/ExperienceCard.tsx]: You could consider moving some of this to a class in your globals.css

You can make a utility class to handle a lot if this:

Globals.css
```css
@layer utilities{
 .Card{
   @apply shadow-work cursor-pointer bg-white-900 etc...etc...
 }
 .Card:hover{
  @apply sm:hover:shadow-work sm:hover:ml-[30px] etc.. etc..
 }
}
```

If you find yourself using a lot of these in groups over and over- that a good way to clean it up. You can also extend your tailwind theme- But regardless I think you might have reached a limit for reasonable amount of tailwind classes in one className 😂

File: constants/projects.json [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+            "color": "#416CEA"
+        },
+        {
+            "img": "https://example.com",
+            "title": "JobIT",
+            "desc": "A Job Finding Application",
+            "frontEnd": ["ReactJS","NextJS"],
+            "backEnd": ["NodeJS", "MongoDB"],
+            "color": "#0BAB7C"
+        }
```
Comment (brandonetter) [inline constants/projects.json:18]: Excellent blocking out data. Now you already know what you'll need in sanity when the integration comes!

**Review by brandonetter — CHANGES_REQUESTED** [review]

File: components/ExperienceCard.tsx (hunk 1 above)
Comment (KamsiyonnaObi) [reply components/ExperienceCard.tsx]: Haha, I knew it was getting out of hand. I'll make custom classes as you suggested.

File: components/ExperienceCard.tsx [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
+            years_worked: string;
+        }[];
+    };
+}
+
+const ExperienceCard = (props: Props) => {
+  return (
+    <>
+        {props.experiences.data.map((exp, idx) => (
+            <Card key={idx} className="Card Card:hover w-[345px] py-[36px] pl-[36px] rounded-[10px] gap-[18px] sm:w-[607px] dark:bg-transparent hover:dark:bg-black-300">
```
Comment (brandonetter) [inline components/ExperienceCard.tsx]:
```suggestion
            <Card key={idx} className="Card w-[345px] py-[36px] pl-[36px] rounded-[10px] gap-[18px] sm:w-[607px] dark:bg-transparent hover:dark:bg-black-300">
```
You shouldn't need the card:hover here

It looks like you're still adding to this PR. Let me know if you're done with this feature now and want me to merge or not :)

**Review by brandonetter — CHANGES_REQUESTED** [review]

**Comment by KamsiyonnaObi** [comment]
I am done now. Ready to merge 👯‍♂️

**Pull request merged**